Makefile

```diff
@@ -1,20 +1,33 @@
 TEST_TARGET := ./tests/
 PY_PACKAGE := packit
-SOURCE_GIT_IMAGE := packit
```

Comment:
It needs to be updated also here https://github.com/packit-service/packit/blob/43857387ec8ef036ab7242c766c33c42ad8964cd/Makefile#L45

Reply:
`make test-in-container` [does not work](https://github.com/packit-service/packit/blob/master/Makefile#L34) currently, we should comment the make target out for now. Just run `tox` directly.